wiki/whitelist-process.md

```diff
@@ -127,6 +129,32 @@ If at any point in the future you want to be removed form the whitelist, you can
 ## What about whitelisting further in the future?
 That's a valid option too. You don't have to whitelist right away
 
+## How much are the rewards my user's receive worth?
+
+Rewards are distributed across all projects equally through a system called <a href="magnitude.html">magnitude</a>. There are 28,750 GRC minted per day as rewards, so your project's users would receive 1/n of these rewards, with n being the total number of whitelisted projects (currently {{ site.data.whitelist.projects.size }}).
+
+Each user will receive a portion of the rewards allocated to your project. Their portion is relative to other crunchers on your project. So, for example, if a user completed 10% of your project's total recent average credit, they would receive 10% of the total reward. As a project admin, you can use any system you like for allocating credit so long as it is reasonable. For example, you can allocate extra credit for the quick turnaround of workunits or for certain types of workunits.
+
+Users can then keep these GRC, sell them via an exchange, donate them, or use them to purchase goods or services from vendors which accept GRC.
+
+The formula for figuring out the daily rewards your users will receive in total is:
+
+`(current GRC value) * (28750/ number of whitelisted projects)`
+
+You can find GRC's current and historical price through an exchange monitoring site like <a href="https://coinmarketcap.com/currencies/gridcoin/" rel="nofollow">CoinMarketCap</a>. Gridcoin does not control the price of GRC, it is determined by market conditions.
+
+## Can I increase the amount of rewards my user's receive?
+
+Absolutely! Increasing the reward amount can incentivize additional crunching on your project and is often much cheaper than purchasing additional computation from cloud and compute providers, as users are often crunching at a loss or break-even. You can do this via the `rainbymagnitude` feature, which allows you to distribute additional rewards to users based on their relative contribution to your project. This will work just as if the amount of GRC allocated to your project the day of your "rain" is +x where x is the amount you are "raining". Note that this does not provide _guaranteed_ compute, but does help incentivize people to compute more.
+
+1. Choose an amount to add to your total incentive. We suggest at least the current total amount of your incentive. So if your project normally receive x GRC per month, make your "rain" x GRC as well.
+2. Let your users know that you will be dispensing additional rewards, the amount of the additional rewards, and when the amount will be "rained". You can do this via an announcement on your site. It's probably a good idea to also post about it on Gridcoin's reddit, discord, and other communities. Giving users a heads up of several weeks will give them time to reconfigure their machines and increase their recent average credit.
```

Comment:
nit: Reddit and Discord should be capitalized

Reply:
fixed